marklogic-data-hub-spark-connector/build.gradle

```diff
@@ -82,22 +82,30 @@ task copyModulesForConnectorFromCoreProject(type: Copy) {
 processResources.dependsOn copyModulesForConnectorFromCoreProject
 
 shadowJar {
-    classifier "all"
+    classifier "pre"
```

Comment:
Let's use "needs-to-be-fixed" to make it very clear that the output of this task is not yet working.